src/renderer/Generics/redesign/consts.ts

```diff
@@ -37,14 +42,16 @@ export interface ClientProps {
 export type NodeAction = 'start' | 'stop' | 'logs' | 'settings';
 export interface NodeOverviewProps {
   name: NodeBackgroundId;
+  displayName?: string;
   title: string;
   info: string;
-  screenType: 'altruistic' | 'client' | 'validator';
+  screenType: 'nodePackage' | 'altruistic' | 'client' | 'validator';
```

Comment:
`altruistic` was the original wording for 2 packages, so maybe we can delete all mentions, and/or modify logic as needed when related to altruistic?

maybe "client" can become "node" for single ones like Farcaster in the future?